packages/vega/test/scenegraphs/bar-time.json

```diff
@@ -207,7 +207,10 @@
                         }
                       ],
                       "x": 0.5,
-                      "y": 200.5
+                      "y": 200.5,
+                      "ariaLabel": "x axis with values Sun Jan 01 2012 00:00:00 GMT-0800 (Pacific Standard Time),Mon Jan 02 2012 00:00:00 GMT-0800 (Pacific Standard Time),Tue Jan 03 2012 00:00:00 GMT-0800 (Pacific Standard Time),Wed Jan 04 2012 00:00:00 GMT-0800 (Pacific Standard Time),Thu Jan 05 2012 00:00:00 GMT-0800 (Pacific Standard Time),Fri Jan 06 2012 00:00:00 GMT-0800 (Pacific Standard Time),Sat Jan 07 2012 00:00:00 GMT-0800 (Pacific Standard Time)",
```

Comment:
We need nicer default aria text descriptions of `Date` objects.

Reply:
Yes. I think this is not possible with Vega right now. We currently use this aria label: `x axis represents values " + domain('xscale')`. The format is `timeFormat(VALUE, timeUnitSpecifier(tbin.units))` but I don't think we have a way to map the values with the formatting function right now. We would need a `map` expression function, right?